python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -227,7 +227,7 @@ def inverse_transform(self, y: cudf.Series) -> cudf.Series:
         ord_label = y.unique()
         category_num = len(self.classes_)
         if self.handle_unknown == 'error':
-            for ordi in ord_label:
+            for ordi in ord_label.values_host:
```

Comment:
why `values_host` as opposed to `values`?

Reply:
I thought that it might be better to take the data to RAM and iterate there instead of iterating on an array stored on GPU that might require a read to GPU for each to extracted element.

Reply:
Both work, but here `ord_label` refer to a `cudf.Series` and `ordi` to an element of that serie. Might be wrong, but to me, iterating over a GPU array require a transfer of each element which might be quite slow.

Reply:
yeah, given what it is doing with the code afterwards it seems like `values_host` is perfectly fine here, you're right